api/lib/domain/usecases/archive-organization.js

```diff
@@ -1,14 +1,9 @@
-const OrganizationToArchive = require('../models/OrganizationToArchive');
+const OrganizationForAdmin = require('../models/OrganizationForAdmin');
 
-module.exports = async function archiveOrganization({
-  organizationId,
-  userId,
-  organizationToArchiveRepository,
-  organizationForAdminRepository,
-}) {
-  const organizationToArchive = new OrganizationToArchive({ id: organizationId });
-  organizationToArchive.archive({ archivedBy: userId });
-  await organizationToArchiveRepository.save(organizationToArchive);
+module.exports = async function archiveOrganization({ organizationId, userId, organizationForAdminRepository }) {
+  const organization = new OrganizationForAdmin({ id: organizationId });
+  organization.archive({ archivedBy: userId });
+  await organizationForAdminRepository.save(organization);
```

Comment:
Le but de cette PR c'était aussi d'enlever l'appel au `get` qui ne servait pas non ? 
On pourrait renvoyer les infos de l'orga dès le `save` avec juste l'id , le archivedBy et archivedAt
```suggestion
  return await organizationForAdminRepository.save(organization);
```

Reply:
en fait il faut refaire le join avec le userid donc je me suis dit c'est plus juste de le garder separer... et comme ca l'action d'archiver n'est pas impactes par un not found.. dans le usecase on voit bien qu'on archive et puis on va chercher l'orga avec le join du user